pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -203,6 +203,26 @@ func (r *ReconcileInfinispan) Reconcile(request reconcile.Request) (reconcile.Re
 		}
 	}
 
+	// Persistent volume claim for custom variables
+	if infinispan.Spec.Container.CustomLibrariesPVName != "" {
+		customPvc := &corev1.PersistentVolumeClaim{}
+		err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: infinispan.Namespace, Name: infinispan.Name + "-custom-libraries"}, customPvc)
```

Comment:
It's better to use `fmt.Sprintf` for this

Reply:
Is this related to performance? I didn't find any clear answer around

Reply:
Not sure that's performance related. It's much better for code reading and avoiding typos

Reply:
Imo `Sprintf` is advantagous when doing something that requires multiple levels of string concatenation, e.g. `SprintF("%s:%s", str1, str2)` which otherwise would be `str1 + ":" + str2`. I think when it's simply adding a suffix, using `+ str` is fine, maybe even more readable. This comes down to preference though, so I'm happy either way for the prefix or suffix case.

Reply:
it was just curiosity, I'm also happy with both the syntax. Changed to sprintf